### DISCUSSION TITLE
[FR] Addings File "Discussions" to REST API

### DISCUSSION THREAD

**Discussion by jhlchu** [civitai/civitai#310] [Ideas]
Feature request for adding "Discussions" as an endpoint for requesting file discussions (comments, replies, images, and generating parameters like prompts and models).
I did this with selenium and it was painful.

Something like how the frontend is querying.
https://civitai.com/api/trpc/commentv2.getInfinite,commentv2.getCount,commentv2.getThreadDetails,image.getConnectionData?batch=1&input={"0":{"json":{"entityId":1234,"entityType":"image","limit":3,"cursor":null},"meta":{"values":{"cursor":["undefined"]}}},"1":{"json":{"entityId":1234,"entityType":"image"}},"2":{"json":{"entityId":1234,"entityType":"image"}},"3":{"json":{"id":1234,"modelId":1212,"reviewId":11111}}}

**Comment by enporial** [comment]
This would be very beneficial.

**Comment by goldeneave** [comment]
did you figure that out, how could we get the discussion data
